### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by dy-lin**
Here are the proposed grading and instructions for milestone 2. If this merge request is approved, I will copy this README to the instructor repo as well! 

I don't see anyone on Asana assigned to review milestone2, so I'll just add some reviewers!

From the teamwork document:
`Developing material for the course`: TA works independently; but must get the final OK from one other member of the teaching team.

### CODE REVIEW

**Comment by dy-lin** [comment]
> + Milestones 2 and 3 are graded as follows:
>   + Troubleshooting is worth 15 points
>   + Pull requests are worth 15 points, based on:
>     + A good title and description. A good title is self-explanatory, and a good description clearly states *what* was changed in the pull request and *why*.
>     + Productive discussion. Both students engaged in discussion in the pull request, and made changes to the document accordingly.

Oops, just saw this on the course webpage: https://stat545.stat.ubc.ca/collaborative-about/
Is this something I can modify or should I just change the grading scheme in the milestone2 `README.md`?

Currently I have a 45 point total for Milestone 2, as there are 24 errors to fix in the troubleshooting document so it would be an awkward point system to scale it down to 15. 

I will update the pull request part to add in the criteria of `a good title and description` and `productive discussion`, but again, I was planning on having 10 pull requests, each worth 2 points.

File: content/collaborative-project/milestone2/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# STAT 545A Collaborative Project: Milestone 2
+
+**Deadline**: Friday, September 25, 2020 at 23:59 PST
+
+**Points**: 45
```
Comment (vincenzocoia) [inline content/collaborative-project/milestone2/README.md:5]: I think we were aiming for 30 points per Milestone, except for the last one that has an extra 10 for a teamwork document.

File: content/collaborative-project/milestone2/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- Student B who writes the explanation must start:
+    - Productive discussion. Both students engaged in discussion in the pull request, and made changes to the document accordingly.
+
+### Grading
+
+- Each 'corrected' code chunk in the troubleshooting document is worth 1 point, for a total of 24 points
+- Student A's pull request per exercise is worth 1 point, and Student B's comments and merging is worth 1 point, for a total of 2 points per exercise, leading to an overall total of 20 points
+- 1 point is given for tagging the release for submission
+
+**GRAND TOTAL:** 45 points
```
Comment (vincenzocoia) [inline content/collaborative-project/milestone2/README.md:93]: Need to change this if we do go with 30 points

**Review by vincenzocoia — COMMENTED** [review]
Nice level of detail given to guide the students! Thank you! We just need to check whether 45 points makes sense or not -- we could move exercises to Milestone 3, or re-scale to 30 points.

**Comment by vincenzocoia** [comment]
Oh! And I just saw your second comment, @dy-lin . I'll take a look at the troubleshooting document for better context.

**Comment by vincenzocoia** [comment]
OK, once the number of points issue is sorted out, I approve this pull request :)

**Review by vincenzocoia — APPROVED** [review]

**Comment by dy-lin** [comment]
Ok-- I think we can change it so that it sticks to the original plan of 30 points, adhering to the original 15/15 system.

Troubleshooting: 15 points (0.625 points per error)
Pull requests: 15 points (1.5 points per pull request: 0.75 for creating the pull request/proposing a fix; 0.75 for explaining the fix)

**Pull request merged**